adapters/lora-vertx/src/main/java/org/eclipse/hono/adapter/lora/providers/FireflyProvider.java

```diff
@@ -76,6 +86,53 @@ public LoraMessageType extractMessageType(final JsonObject loraMessage) {
         if (loraMessage.containsKey(FIELD_FIREFLY_FPORT)) {
             returnMap.put(LoraConstants.APP_PROPERTY_FUNCTION_PORT, loraMessage.getInteger(FIELD_FIREFLY_FPORT));
         }
+        if (loraMessage.getJsonObject(FIELD_FIREFLY_SERVER_DATA, new JsonObject())
```

Comment:
I think the code would be easier to read, if 
`loraMessage.getJsonObject(FIELD_FIREFLY_SERVER_DATA, new JsonObject())` 
got extracted into a variable and subsequent occurrences got replaced with the variable.
Same goes for `loraMessage.getJsonObject(FIELD_FIREFLY_PARSED_PACKET, new JsonObject())`.

Reply:
done